apps/prairielearn/src/lib/question-render.js

```diff
@@ -602,7 +589,6 @@ export async function renderPanelsForSubmission({
   /** @type {SubmissionPanels} */
   const panels = {
     submissionPanel: null,
-    scorePanel: null,
```

Comment:
This value was never used anywhere. The score panels are actually `questionScorePanel` and `assessmentScorePanel`.